assets/models/system/microsoft-rad-dino/description.md

```diff
@@ -0,0 +1,253 @@
+...     patches_grid = rearrange(flat_tokens, "b (h w) c -> b c h w", h=embeddings_size)
+...     return patches_grid
+...
+>>> flat_patch_embeddings = outputs.last_hidden_state[:, 1:]  # first token is CLS
+>>> reshaped_patch_embeddings = reshape_patch_embeddings(flat_patch_embeddings)
+>>> reshaped_patch_embeddings.shape  # (batch_size, num_channels, height, width)
+torch.Size([1, 768, 37, 37])
+```
+
+## Training details
+
+### Training data
+
+<!-- This should link to a Dataset Card, perhaps with a short stub of information on what the training data is all about as well as documentation related to data pre-processing or additional filtering. -->
+
+We used images from five public, deidentified chest X-ray datasets to train this checkpoint of RAD-DINO.
+
+| Dataset   | Num. images |
+| --------- | ----------: |
+| [MIMIC-CXR](https://www.nature.com/articles/s41597-019-0322-0) | 368 960 |
+| [CheXpert](https://ojs.aaai.org/index.php/AAAI/article/view/3834) | 223 648 |
+| [NIH-CXR](https://openaccess.thecvf.com/content_cvpr_2017/html/Wang_ChestX-ray8_Hospital-Scale_Chest_CVPR_2017_paper.html) | 112 120 |
+| [PadChest](https://www.sciencedirect.com/science/article/abs/pii/S1361841520301614) | 136 787 |
+| [BRAX](https://www.nature.com/articles/s41597-022-01608-8) | 41 260 |
+| **TOTAL** | 882 775 |
+
+Images in the validation and test sets used to train [MAIRA](https://arxiv.org/abs/2311.13668) were excluded from the training set of RAD-DINO.
+The list of image files used for training is available at [`./training_images.csv`](./training_images.csv).
+
+Note this checkpoint is different from the one in the paper, where some private data was used (and fewer GPUs).
+The checkpoint shared here is trained for 35 000 iterations (the total number of iterations in the run was 100 000, but we selected this checkpoint using linear probing on the validation sets of the evaluation datasets described in the paper).
+We used 16 nodes with 4 A100 GPUs each, and a batch size of 40 images per GPU.
+
+### Training procedure
+
+<!-- This relates heavily to the Technical Specifications. Content here should link to that section when it is relevant to the training procedure. -->
+
+We refer to the [manuscript](https://arxiv.org/abs/2401.10815) for a detailed description of the training procedure.
+
+#### Preprocessing
+
+All DICOM files were resized using B-spline interpolation so that their shorter size was 518, min-max scaled to [0, 255], and stored as PNG files.
+
+#### Training hyperparameters
+
+- **Training regime:** fp16 using PyTorch-FSDP mixed-precision.
+
+<!--fp32, fp16 mixed precision, bf16 mixed precision, bf16 non-mixed precision, fp16 non-mixed precision, fp8 mixed precision -->
+
+## Evaluation
+
+<!-- This section describes the evaluation protocols and provides the results. -->
+
+Our evaluation is best described in the [manuscript](https://arxiv.org/abs/2401.10815).
+
+<!-- ### Testing data, factors & metrics
```

Comment:
This section is just a bunch of comments. Suggestions to remove it, else will look incomplete and odd